src/main/java/org/llorllale/cactoos/matchers/Throws.java

```diff
@@ -66,35 +82,76 @@
      * @param msg The expected exception message.
      * @param type The expected exception type.
      */
-    public Throws(final String msg, final Class<? extends Exception> type) {
+    public Throws(
+        final String msg, final Class<? extends Exception> type
```

Comment:
@vzurauskas  argument `type`  is named the same as `private final type` such name can be confusing

Reply:
@ilyakharlamov Using actual field name as constructor parameter [is a commonly accepted Java practice](https://stackoverflow.com/questions/991757/best-practice-for-parameter-naming-in-java-constructors-and-simple-setters). Is it different for this project? Qulice doesn't seem to mind.

Reply:
@vzurauskas is this why you disabled `MemberNameCheck` and `ParameterNameCheck`?

I agree with your sentiment - I also don't mind parameters with the same names as members in our ctors (because our philosophy is to have them small). But now you're needlessly including a change in rules in this PR. I suggest you open a ticket for this and we can discuss there.

Reply:
@llorllale They don't check variable-field name hiding. They check variable name length. I disabled them with the intention that they will be re-enabled as soon as the new `todo` is done and `messageExpectation` and `expectationDescription` fields are shortened. I probably should have included that in the `todo` description.